test/cli.test.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+var test = require('tape');
+var path = require('path');
+var fs = require('fs');
+var os = require('os');
+var cli = path.resolve(__dirname, '..', 'bin', 'vtshave.js');
+var spawn = require('child_process').spawn;
+
+var tile = path.join(__dirname, 'fixtures', 'tiles', 'sf_16_10465_25329.vector.pbf');
+var style = path.join(__dirname, 'fixtures', 'styles', 'bright-v9.json');
+
+
+if (process.env.TOOLSET && process.env.TOOLSET === 'asan') {
```

Comment:
This runs for all jobs except the `asan` job. Because getting asan working for subprocesses is impossible on OS X.